src/includes/content/wizard/affliate/flycricket.pug

```diff
@@ -1,15 +1,5 @@
-#fc-container.box
```

Comment:
After reading your other comments and thinking about this a little further, I removed the flycricket box from being visible, but left the form, which is still needed for sending data to flycricket.

In this update, the deploy can _only_ be triggered from the modal. My thought process is:

The two end actions for the user are 'copy' or 'deploy'. In the existing flow, these were on separate pages (step3 vs modal). In this commit, all actions for the user are in one place so the user can easily decide what to do with the generated policy/terms. Plus it cleans up your step3 to direct the users right to the generated content.

What do you think @nisrulz? I think this'll increase deployments, so I might be a little biased :) but I do think it's a smoother flow, especially since terms have been added to step3.